org-node/src/lib.rs

```diff
@@ -372,6 +381,58 @@ async fn subscribe_events(url: String, addresses: Vec<Address>, update: mpsc::Se
     }
 }
 
+/// Read named pipe / fifo file;
+async fn sign_refs(root_dir: Option<PathBuf>, mut handle: client::Handle) {
```

Comment:
Would be a bit less confusing if this function was called the same thing as the command it uses, ie. `update_refs`

Reply:
514bf38 updates.